File: springboot-mustache-crud/src/main/java/com/knf/dev/demo/springbootjpamustachecrud/repository/UserRepository.java
package com.knf.dev.demo.springbootjpamustachecrud.repository;

import com.knf.dev.demo.springbootjpamustachecrud.model.User;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface UserRepository extends CrudRepository<User, Long> {
}

File: kotlin-spring-main/kotlinspringjpagroovycrudexample/src/main/kotlin/com/knf/dev/demo/kotlinspringbootcrudexample/repository/UserRepository.kt
package com.knf.dev.demo.kotlinspringbootcrudexample.repository

import com.knf.dev.demo.kotlinspringbootcrudexample.model.User
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface UserRepository : CrudRepository<User?, Long?>